README.rst

```diff
@@ -17,15 +17,46 @@
 .. image:: https://img.shields.io/github/tag/fair-data-austria/invenio-madmp.svg
         :target: https://github.com/fair-data-austria/invenio-madmp/releases
 
-.. image:: https://img.shields.io/pypi/dm/invenio-madmp.svg
-        :target: https://pypi.python.org/pypi/invenio-madmp
-
 .. image:: https://img.shields.io/github/license/fair-data-austria/invenio-madmp.svg
         :target: https://github.com/fair-data-austria/invenio-madmp/blob/master/LICENSE
 
-Invenio module for maDMP integration.
+.. image:: https://img.shields.io/badge/code%20style-black-000000.svg
+    :target: https://github.com/psf/black
+
+
+Invenio module for integrating machine-actionable data management plans (maDMPs)
+according to the `Research Data Alliance Common Standard <https://github.com/RDA-DMP-Common/RDA-DMP-Common-Standard>`_.
+
+
+::
+
+  I can help
+        -- Android Minsky
```

Comment:
```suggestion
```

Reply:
I removed the quote along with the reST start-of-quote marker.